### DISCUSSION TITLE
Virus detected in pre-compiled Windows software for NanoVNA

### DISCUSSION THREAD

**Discussion by BeamPower** [jankae/LibreVNA#265] [General]
VirusTotal detected a virus on the pre-compiled Windows software offered on this page:

https://nanovna.com/?page_id=141

In the 7z file:
http://nanovna.com/wp-content/uploads/2021/11/NanoVNA-App_1_1_208_6.3G.7z

MaxSecure
Trojan.Malware.300983.susgen
1/64 security vendor flagged this file as malicious

Same virus detected in NanoVNA-App.exe after unzipping the 7z file.

**Comment by jankae** [comment]
This is the repository for the LibreVNA and it has nothing to do with the NanoVNA please report this somewhere else.
